qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -571,31 +642,45 @@ def new_uhrig_sequence(
 def new_periodic_sequence(
     duration=None, number_of_offsets=None, pre_post_rotation=False, **kwargs
 ):
-    """
-    Creates Periodic Single Axis Sequence.
+    r"""
+    Creates the periodic sequence.
 
     Parameters
     ---------
     duration : float
-        Total duration of the sequence. Defaults to None
+        Total duration of the sequence :math:`\tau`. Defaults to None.
     number_of_offsets : int, optional
-        Number of offsets. Defaults to None
+        Number of offsets :math:`n`. Defaults to None.
     pre_post_rotation : bool, optional
-        If True, a :math:`\\pi.2` rotation is added at the
+        If True, a :math:`X_{\pi/2}` rotation is added at the
         start and end of the sequence.
     kwargs : dict
-        Additional keywords required by
-        qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
+        Additional keywords required by DynamicDecouplingSequence.
 
     Returns
     -------
-    qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
-        Periodic (single-axis) sequence
+    DynamicDecouplingSequence
+        The periodic sequence.
 
     Raises
     ------
     ArgumentsValueError
         Raised when an argument is invalid.
+
+    Notes
+    -----
+    The periodic sequence [#]_ is parameterized by duration :math:`\tau` and number of
+    offsets :math:`n`. The sequence consists of :math:`X_{\pi}` operations at offsets given by
+
+    .. math::
+        t_i = \frac{\tau}{n + 1} \;,
+
+    where :math:`i = 1, \cdots, n`.
+
+    References
+    ----------
+    .. [#] `Lorenza Viola and Emanuel Knill, Physical Review Letters 90, 037901 (2003).
```

Comment:
```suggestion
    .. [#] `L. Viola and E. Knill, Physical Review Letters 90, 037901 (2003).
```

Oops, I forgot to abbreviate these names last time